vault/resource_managed_keys_test.go

```diff
@@ -158,6 +158,53 @@ func TestManagedKeys(t *testing.T) {
 	})
 }
 
+// The following test requires a Vault server to be set up with a specific server configuration
```

Comment:
Are these steps documented somewhere? I am not confident I could get this test running by following this comment alone.

Reply:
Good call, I can document the steps to create a PKCS#11 key and share it to our Google Drive folder or Confluence

Reply:
Thanks! I made a note to discuss in our dev sync. But I think we should make sure any local only test setups are well documented. I think creating a Confluence page to document these is a good idea!